frames/src/main/java/com/checkout/frames/component/base/InputComponentState.kt

```diff
@@ -9,5 +9,6 @@ internal data class InputComponentState(
     val subtitleState: TextLabelState = TextLabelState(),
     val infoState: TextLabelState = TextLabelState(),
     val errorState: TextLabelState = TextLabelState(),
-    val isInputFieldOptional: Boolean = false
+    val isInputFieldOptional: Boolean = false,
+    val isInputFieldHidden: Boolean = false
```

Comment:
I don't see need for isHidden, it was disscussed from the start that if style is null then component not shown.

Reply:
Removed as discussed.

Reply:
@chintan-soni-cko Looks like it wasn't removed.